lib/icecast.js

```diff
@@ -1,37 +1,45 @@
-var request = require('request');
-var parseXmlString = require('xml2js').parseString;
-var urlParser = require('url');
+var request = require("request");
+var parseXmlString = require("xml2js").parseString;
+var urlParser = require("url");
 var utils = require("./utils.js");
 
 function getIcecastStation(url, callback) {
   var urlObject = urlParser.parse(url);
-  var icecastJsonUrl = urlObject.protocol + "//" + urlObject.hostname + ":" + urlObject.port + "/status-json.xsl"
+  var icecastJsonUrl =
+    urlObject.protocol +
+    "//" +
+    urlObject.hostname +
+    ":" +
+    urlObject.port +
+    "/status-json.xsl";
 
-  var res = request({
-    url: icecastJsonUrl
-  }, function(error, response, body) {
+  var res = request(
+    {
+      url: icecastJsonUrl
+    },
+    function(error, response, body) {
+      if (error) {
+        return callback(error);
+      }
 
-    if (error) {
-      return callback(error);
-    }
-
-    if (response.statusCode !== 200) {
-      return callback(new Error("HTTP error."))
-    }
+      if (response.statusCode !== 200) {
+        return callback(new Error("HTTP error."));
+      }
 
-    res.on('error', function(error) {
-      res.abort();
-      return callback(error);
-    });
+      res.on("error", function(error) {
```

Comment:
I know I'm totally not consistent myself, but it seems like we should probably standardize on single quotes, since somebody told me that's what most Javascript people use?  I'm not sure how that was decided, but I'll go with it.

Reply:
Stupid prettier. I'll have to change my prettier setting to flip over. But your correct I tend to always use single quotes.

Reply:
Done. Although it changed ALL double quotes to singles lol